apps/trading-e2e/src/integration/trading-orders.cy.ts

```diff
@@ -29,50 +30,49 @@ describe('orders list', { tags: '@smoke' }, () => {
     cy.wait('@Orders').then(() => {
       expect(subscriptionMocks.OrdersUpdate).to.be.calledOnce;
     });
+    cy.wait('@Markets');
   });
-  it('renders orders', () => {
+  it.only('renders orders', () => {
```

Comment:
chore (blocking): Theres only one test in this describe block but probably worth removing the `.only`